torchvision/prototype/transforms/_augment.py

```diff
@@ -180,3 +184,190 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
+
+    def _copy_paste(
+        self,
+        image: Any,
+        target: Dict[str, Any],
+        paste_image: Any,
+        paste_target: Dict[str, Any],
+        blending: bool = True,
+        resize_interpolation: F.InterpolationMode = F.InterpolationMode.BILINEAR,
+    ) -> Tuple[Any, Dict[str, Any]]:
+
+        # Random paste targets selection:
+        num_masks = len(paste_target["masks"])
+
+        if num_masks < 1:
+            # Such degerante case with num_masks=0 can happen with LSJ
+            # Let's just return (image, target)
+            return image, target
+
+        # We have to please torch script by explicitly specifying dtype as torch.long
+        random_selection = torch.randint(0, num_masks, (num_masks,), device=paste_image.device)
+        random_selection = torch.unique(random_selection).to(torch.long)
+
+        paste_masks = paste_target["masks"][random_selection]
+        paste_boxes = paste_target["boxes"][random_selection]
+        paste_labels = paste_target["labels"][random_selection]
+
+        masks = target["masks"]
+
+        # We resize source and paste data if they have different sizes
+        # This is something different to TF implementation we introduced here as
+        # originally the algorithm works on equal-sized data
+        # (for example, coming from LSJ data augmentations)
+        size1 = image.shape[-2:]
+        size2 = paste_image.shape[-2:]
+        if size1 != size2:
+            paste_image = F.resize(paste_image, size1, interpolation=resize_interpolation)
+            paste_masks = F.resize(paste_masks, size1, interpolation=F.InterpolationMode.NEAREST)
+            # resize bboxes:
+            ratios = torch.tensor((size1[1] / size2[1], size1[0] / size2[0]), device=paste_boxes.device)
+            paste_boxes = paste_boxes.view(-1, 2, 2).mul(ratios).view(paste_boxes.shape)
+
+        paste_alpha_mask = paste_masks.sum(dim=0) > 0
+
+        if blending:
+            paste_alpha_mask = F.gaussian_blur(
+                paste_alpha_mask.unsqueeze(0),
+                kernel_size=(5, 5),
+                sigma=[
+                    2.0,
+                ],
+            )
+
+        # Copy-paste images:
+        image = (image * (~paste_alpha_mask)) + (paste_image * paste_alpha_mask)
+
+        # Copy-paste masks:
+        masks = masks * (~paste_alpha_mask)
+        non_all_zero_masks = masks.sum((-1, -2)) > 0
+        masks = masks[non_all_zero_masks]
+
+        # Do a shallow copy of the target dict
+        out_target = {k: v for k, v in target.items()}
+
+        out_target["masks"] = torch.cat([masks, paste_masks])
+
+        # Copy-paste boxes and labels
+        boxes = masks_to_boxes(masks)
+        out_target["boxes"] = torch.cat([boxes, paste_boxes])
+
+        labels = target["labels"][non_all_zero_masks]
+        out_target["labels"] = torch.cat([labels, paste_labels])
+
+        # Update additional optional keys: area and iscrowd if exist
```

Comment:
Is this an actual `TODO`? I don't think we can unless we make them attributes or the like on `features.BoundingBox`. We can easily compute `area` at runtime, i.e. we could have a `.area()` method. 

`is_crowd` will be a lot harder since it is metadata that is associated with every box individually. Meaning , a `BoundingBox.shape == (10, 4)` with a boolean `.is_crowd is True` makes little sense. A naive way would be to make the `.is_crowd` attribute a list or tuple, but than we get into hot waters whenever an op changes the shape of the tensor. For example, indexing would effect the tensor data, but would leave the `.is_crowd` attribute as is.

My suggestion is to remove all these occurrences until we have figured out if and how we want to handle them.